src/module/migration/migrations/835-initiative-longform.ts

```diff
@@ -0,0 +1,21 @@
+import { ActorSourcePF2e } from "@actor/data";
+import { SKILL_ABBREVIATIONS, SKILL_DICTIONARY } from "@actor/values";
+import { setHasElement } from "@util";
+import { MigrationBase } from "../base";
+
+/** Converts sheet selected skills from shortform (ath) to longform (athletics) */
+export class Migration835InitiativeLongform extends MigrationBase {
+    static override version = 0.835;
+
+    override async updateActor(actor: ActorSourcePF2e): Promise<void> {
+        if (!("initiative" in actor.system.attributes)) return;
+
+        const initiative = actor.system.attributes.initiative;
+        if (!initiative || !("ability" in initiative)) return;
+
+        const ability = initiative.ability;
+        if (setHasElement(SKILL_ABBREVIATIONS, ability)) {
```

Comment:
Would this be cleaner as:
```
if (ability in SKILL_DICTIONARY) {
    initiative.ability = SKILL_DICTIONARY[ability];
}
```

Reply:
SKILL_DICTIONARY is a set, and `in` doesn't work for sets.

Reply:
Ah wait I see I misread. This works, but I suspect this will cause issues in the future when we finally promote longforms to the true way.

Reply:
It won't cue typescript that the key is legit